evaluation/user-feedback.mdx

```diff
@@ -1,60 +0,0 @@
-
```

Comment:
Why was this file deleted?

Reply:
Yes, we want to keep it so we have the menu item that might be exactly what a user is looking for.

Reply:
how about like so? I made a "snippet" out of the content of the page and put it in both monitoring/collect-user-feedback and to restored evaluation/user-feedback. this way we have both pages but they just display the same thing